pycrostates/viz/cluster_centers.py

```diff
@@ -114,6 +142,24 @@ def plot_cluster_centers(
             ax = axes[i, j]
         # plot
         plot_topomap(center, info, axes=ax, show=False, **kwargs)
+        # Add min max vector
+        if show_gradient:
+            i_min = np.argmin(center)
+            i_max = np.argmax(center)
+            pos = mne.channels.layout._find_topomap_coords(info, picks="all")
```

Comment:
Maybe import only this function at the top-level? Or even import it directly in this `if` condition since it's very specific.